imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -134,13 +135,45 @@ def convert_counts_to_rate(data: np.ndarray, acq_duration: int) -> npt.NDArray:
     return count_rate.astype(np.float64)
 
 
-def calculate_calibration_factor(time: int) -> None:
+def read_in_flight_cal_data() -> pd.DataFrame:
+    """
+    Read in-flight calibration data.
+
+    In-flight calibration data file will contain rows where each line
+    has 8 numbers, with the first being a time stamp in MET, and the next
+    7 being the factors for the 7 detectors.
+
+    This file will be updated weekly with new calibration data. In other
+    words, one line of data will be added each week to the existing file.
+    File will be in CSV format. Processing won't be kicked off until there
+    is in-flight calibration data that covers science data. TODO: decide
+    filename convention given this information. This function is a
+    placeholder for reading in the calibration data until we decide on
+    how to read calibration data through dependencies list.
```

Comment:
```suggestion
    is in-flight calibration data that covers science data.
    
    TODO: decide filename convention given this information. This function is a
    placeholder for reading in the calibration data until we decide on
    how to read calibration data through dependencies list.
```

Just so the TODO is not buried within the paragraph and can be seen more easily